### DISCUSSION TITLE
How to wrap all screens in a component from a single place

### DISCUSSION THREAD

**Discussion by mohsinulhaq** [software-mansion/react-native-screens#837] [Q&A] [answered]
I have a native stack navigator setup in place, and now I want to wrap all of those screens in an ImageBackground from a single place. Is it possible? Maybe I missed it, but I couldn't find anything relevant in the docs.

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @mohsinulhaq,
I'm worried that's not possible. The solution is to wrap the content of every screen one by one with `ImageBackground`. A short explanation would be that the navigator decides about the navigation between the screens, but not about what is currently displayed.

**Reply by mohsinulhaq** [reply]
Oh, that's sad. Thanks for the quick response though. Much appreciated.
